messages/en.json

```diff
@@ -112,14 +112,9 @@
   "admin/pages.admin.tabs.redirects": "Redirects",
   "admin/pages.editor.component-list.action-menu.delete": "Delete",
   "admin/pages.editor.component-list.action-menu.reset": "Reset",
-  "admin/pages.editor.component-list.button.save": "Save",
-  "admin/pages.editor.component-list.button.undo": "Undo",
-  "admin/pages.editor.component-list.modal.button.cancel": "Discard",
-  "admin/pages.editor.component-list.modal.button.save": "Save",
-  "admin/pages.editor.component-list.modal.text": "You have unsaved modifications",
-  "admin/pages.editor.component-list.toast.error": "Could not be saved.",
-  "admin/pages.editor.component-list.toast.success": "Saved successfully.",
+  "admin/pages.editor.component-list.action-menu.tooltip": "This card is currently being edited.",
```

Comment:
IMHO, we should stick to "content", since it's more of a semantic description, while "card" is kind of purely morphological; perhaps @ntmedina could tell us a bit about her motivations to choose the latter.

Reply:
Although I like "content", the problem is this is against what Geraldo, Guilherme, Afonso and Sávio are advocating, which is calling these block configurations.

Reply:
I'd rather use "block configurations" than "content cards".

Reply:
Hey guys, everything is a block configuration, like everything is an app/component when use react. To differs content from layout (that is a block configuration too) we create the concept of content, that is unique, individual. Users can create any content they want for all the contexts they need, it's important to differs content because it's a block configuration that users edit all the time. We do have to continue the layout vs content editing research, but for now, that we don't have the template editor, and we are not focusing on that, I dind't say this is a problem. Peu gave the idea of using `content cards`, because our users are familiar with this visual clue, but discussing more we decide that we'll use something more generic that all users, even new users can understand. So, yoou can use: `Content list`.